setup.py

```diff
@@ -5,8 +5,8 @@
 setup(
     name='algorithmia-adk',
     version=os.environ.get('ADK_VERSION', '0.0.0'),
-    description='adk python ADK client',
-    long_description='adk Development Kit code used for creating Python algorithms on adk. Built into the Algorithmia client',
+    description='Algorithmia Python ADK client',
+    long_description='Algorithm Development Kit code used for creating Python algorithms on adk. Built into the Algorithmia client',
```

Comment:
Looking at the long description now, we should replace `on adk` with `on Algorithmia`

Reply:
Yes I'd prefer that too